apps/studio/src/store/modules/TabModule.ts

```diff
@@ -78,6 +86,18 @@ export const TabModule: Module<State, RootState> = {
       context.commit('set', [])
       context.commit('setActive', null)
     },
+    async reopenLastClosedTab(context) {
+      for (let i = context.state.lastClosedTabs.length - 1; i >= 0; i--) {
+        const tab = context.state.lastClosedTabs[i]
+        // Does this tab belong to the current connection?
```

Comment:
Do you also have to check workspace id?

Reply:
The `OpenTab` here has still been removed from the database and doesn't have an ID, so it still needs saving.

Also -- what happens with the position of the tab? Does it go back to the right place?

Reply:
I'll look into the workspace id and the position of the tab.

I was under the impression that we want this to behave like vscode, so no need to save `lastClosedTabs` to database. But I just realized that chrome persists the closed tabs even though I exit the program. It works too.

Reply:
Should we even not remove tabs from database and just mark it as "closed"?